timon/run.py

```diff
@@ -133,7 +133,11 @@ async def run_once(options, first=False, cfg=None):
     t_delta_next = max(t_next - t, 0)
 
     if runner.notifier_objs:
+        # rem: `notifier_objs` is the list of notifiers
```

Comment:
what does 'rem:' stand for?

'remark' ? 'reminder?' does it add any value?

Reply:
you ever wrote some old cmd batch scripts :3 that's a habit I picked from there where comments are written with the `rem` "command", like 
```batch
cd somehere
rem this a comment
mkdir blabla
```